src/Sulu/Bundle/ContentBundle/Preview/Preview.php

```diff
@@ -142,8 +142,13 @@ public function updateProperty($userId, $contentUuid, $webspaceKey, $locale, $pr
             throw new PreviewNotFoundException($userId, $contentUuid);
         }
 
-        $content = $this->update($userId, $webspaceKey, $locale, $property, $data, $content);
-        $this->previewCache->saveStructure($content, $userId, $contentUuid, $webspaceKey, $locale);
+        try {
+            $content = $this->update($userId, $webspaceKey, $locale, $property, $data, $content);
+        } catch (\Twig_Error $ex) {
+            throw new TwigPreviewException($ex);
+        } finally {
```

Comment:
Does this really make sense? Can you save something in the cache even if there has been a failure?